examples/single/link.json

```diff
@@ -15,7 +15,7 @@
     "roadway": "residential",
     "length": 80.935,
     "maxspeed": "",
-    "name": "Arkwright Street,North Rivoli Street",
+    "name": "Arkwright Street",
```

Comment:
Why shorten the name?

Reply:
Just to differentiate from the larger network when I'm reading errors (this net is really not "real")